specification.md

```diff
@@ -4537,7 +4536,7 @@ If the start definition is of type `object`, it has the following structure:
 ```json
 {
   "stateName": "MyStartingstate",
-  "schedule": "2020-03-20T09:00:00Z/2020-03-20T15:00:00Z"
+  "schedule": "2020-03-20T09:00:00Z/PT2H"
```

Comment:
Can you explain why this is needed? This is  mean to be ISO_8601
<start>/<end> time duration that tells runtimes the start and end times when instances should be created.

Maybe if you wanted we could do a separate sample that does <start>/<duration>, that would be ok too if you wanted.

Reply:
Ok nice catch,  if its not there then let's add it (and maybe say that its primary use is within schedule property).

Reply:
start/end is the time interval 
Rn / start / end would be repeating interval

I would change all "R/...." with "Rn/...." to show that repeating is optional

Reply:
But again, the spec already says that

> Unlike full ISO 8601, this derivative format does not allow expression of an explicit number of recurrences

So basically Rn, e.g. R5/...., is not supported - so we cannot specify bounded interval.
Do you want me to rewrite it also for Rn?

Reply:
Added start/end use case. Do you also want me to add R[n] to the spec? Then the ISO 8601 spec says:
"Leaving out the value for [n] or specifying a value of -1, means an unbounded number of repetitions. A value of 0 for [n] means the interval is not repeated. "
Then it is unclear to me how we should interpret it. If R0 means interval is not repeated, so we have just one interval - then we should start one workflow at its beginning? With R1 we will repeat interval once, so there will be 2 intervals specified - and at each interval's start we will execute a workflow, so there will be 2 workflows triggered?

Then for `R[n]/<Duration>/<End>` we will execute it n+1 times, last one executed "duration" before "end"?

Reply:
@tsurdilo I think we can at least leave start/end use case as it is straight-forward, just let me know what you think about it.

Reply:
ISO 8601 has been reverted. Just examples were updated to show a correct possibility of interval syntax (no start/end).